lib/reporter/alb_errors_report_builder.rb

```diff
@@ -91,10 +91,10 @@ def empty_5_minute_period_hash
 
     def twenty_four_hours_ago
       @twenty_four_hours_ago ||= begin
-        time = 24.hours.ago
+        time = (TIME_ZONE.now - 24.hours)
```

Comment:
@jmhooper I guess, one downside to a constant like this is that the `.ago`'s don't work as expected, so I had to make them more explicit. This is risky/dangerous/confusing so if we don't explicitly set `Time.zone` to `nil` or something that would blow up, we could get confusing stuff down the line.

If you think it's not worth doing, I can just close this out? Would love your thoughts/feedback here

Reply:
I think it is alright. Ideally we'd get these running in CircleCI which should use UTC and catch this sort of thing.

Reply:
Setting up Circle and CodeClimate for this repo is on my todo list. I can also open a ticket for that

Reply:
Yeah CircleCI would be a big win

Reply:
Ok then this sounds good and I'll merge this for now